core/src/main/resources/logback.xml

```diff
@@ -6,7 +6,7 @@
     <logger name="org.springframework.statemachine.support" level="ERROR"/>
     <logger name="com.sequenceiq.ambari.client.AmbariClientUtils" level="ERROR"/>
     <logger name="org.apache.kafka.clients.producer" level="ERROR"/>
-    <logger name="com.amazonaws.request" level="DEBUG"/>
+    <!-- <logger name="com.amazonaws.request" level="DEBUG"/> -->
```

Comment:
do we want to keep this? if yes we could create a new appender with anonymization enabled on that log

Reply:
I don't want to do that. We send a lot of requests to AWS and it has a serious performance drawback to anonymize all the requests.